### ISSUE TITLE
make docker creates bin directory as root

### ISSUE THREAD

**Issue by brijshah** [awslabs/amazon-ecr-credential-helper#62]
after running `make docker`, I get the following error:
```
docker run --rm \
-e TARGET_GOOS= \
-e TARGET_GOARCH= \
-v /home/ec2-user/amazon-ecr-credential-helper/bin:/go/src/github.com/awslabs/amazon-ecr-credential-helper/bin \
sha256:50f9ca61f0f958764f8f4b08067aaeeea762c85c9e318cec883f4a56b672287f
. ./scripts/shared_env && ./scripts/build_binary.sh ./bin/local
mkdir: cannot create directory './bin/local': Permission denied
go install github.com/awslabs/amazon-ecr-credential-helper/ecr-login/cli/docker-credential-ecr-login: mkdir ./bin/local/: permission denied
make: *** [bin/local/docker-credential-ecr-login] Error 1
Makefile:39: recipe for target 'bin/local/docker-credential-ecr-login' failed
make: *** [docker] Error 2
```

the bin folder that gets created is created as root, adding `--privileged=true` to the docker command inside the make file seems to work, however I dont think thats correct.

**Comment by samuelkarp** [comment]
I think we could fix this by just pre-creating the bin folder in the Makefile instead of in the container.  I'd be happy to accept a pull request that fixes this.

**Comment by wingyplus** [comment]
This problem is still exists for you? @brijshah

**Comment by samuelkarp** [comment]
Fixed with #119.
